tests/testthat/setup-testing_depth.R

```diff
@@ -0,0 +1,52 @@
+#' Returns testing depth set by an environmental variable.
+#'
+#' @details Looks for the option `TESTING_DEPTH` first, if not set,
+#' takes the system environmental variable `TESTING_DEPTH`. If neither
+#' is set, then returns 3 by default. If the value of `TESTING_DEPTH`
+#' is not a scalar numeric, then returns 3.
+#'
+#' @return `numeric(1)` the testing depth.
+#'
+get_testing_depth <- function() {
+  default_depth <- 3
+  depth <- getOption("TESTING_DEPTH", Sys.getenv("TESTING_DEPTH", default_depth))
+  depth <- tryCatch(
+    as.numeric(depth),
+    error = function(error) default_depth,
+    warning = function(warning) default_depth
+  )
+
+  if (length(depth) != 1) {
+    depth <- default_depth
+  }
```

Comment:
```suggestion
  if (length(depth) != 1) depth <- default_depth
```

Reply:
styler will revert this anyway

Reply:
I always prefer `{}`. Even when it's one liner. But, I'll accept it. I've seen our repos have this.

Reply:
https://github.com/insightsengineering/teal/pull/1161#discussion_r1531884490

Reply:
thanks @vedhav